CRM/Upgrade/Incremental/sql/6.2.alpha1.mysql.tpl

```diff
@@ -1 +1,4 @@
 {* file to handle db changes in 6.2.alpha1 during upgrade *}
+
+{* Remove EntityFile records for custom fields; they are redundant with the value of the field *}
+DELETE FROM civicrm_entity_file WHERE entity_table LIKE "civicrm_value_%";
```

Comment:
* Should probably do one of those snapshot thingies.
* On a database with a huge number of file entries this might be slow. I wonder if either it should be one of those task loops, like when the relationshipcache was first populated, or if it should skip it and tell you to do it manually if it you have a lot of entries. I don't actually know that it would be slow, just it uses `like`, and then deleting a large number of records sometimes can be.

Reply:
Ok I've added the snapshot thingie.

I'm not sure why it would be slow. The LIKE operator doesn't start with a wildcard and that's usually what slows it down.

Reply:
Any wildcard will slow it down if it can't use an index, but it's also the deletion of the rows themselves. I'll see if I can try it out somewhere big.

Reply:
Sorry this PR is still on my list, just haven't been able to review it yet this week.